### ISSUE TITLE
Golang operator docker buildx command : missing context path 

### ISSUE THREAD

**Issue by gmtstephane** [operator-framework/operator-sdk#6223]
The command make `docker-buildx` is not working due to missing context parameter 

## Expected Behavior
The make target `make docker-buildx ` works as expected and build the multi-platform docker image 
## Current Behavior
```make
$ make docker-buildx
...
docker buildx create --name project-v3-builder
project-v3-builder
docker buildx use project-v3-builder
docker buildx build --push --platform=linux/arm64,linux/amd64,linux/s390x,linux/ppc64le --tag ghcr.io/gmtstephane/kubelab-networkmanager:latest -f Dockerfile.cross
ERROR: "docker buildx build" requires exactly 1 argument.
```

## Possible Solution
Add the current context path `.` at the end of the command, simmilarly to the ansible template 

https://github.com/operator-framework/operator-sdk/blob/cbeec475e4612e19f1047ff7014342afe93f60d2/testdata/ansible/memcached-operator/Makefile#L100

### Operator SDK
```
operator-sdk version: "v1.25.3", commit: "5779ad7c8901c1e48cf324cec430c26212684b45", kubernetes version: "v1.25.0", go version: "go1.19.3", GOOS: "darwin", GOARCH: "arm64"
```
### Docker 
```
docker version
Server: Docker Desktop 4.14.1 (91661)
```

**Comment by varshaprasad96** [comment]
In order to fix this issue, we need to pull the respective upstream KB version that has the fix.

**Comment by everettraven** [comment]
Closing this issue as this should be resolved as of the v1.28.0 release. If you are still encountering this issue using v1.28.0 or feel this issue was closed in error, feel free to reopen.
